plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -183,18 +193,18 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 			};
 		},
 
-		_handleCodeMirrorChange: function () {
-			var newData = this.codeMirrorEditor.getValue();
-			var preview = this.dialog
-				.getContentElement('main', 'preview')
-				.getElement();
-
-			var iframe = preview.findOne('iframe');
-			if (iframe && iframe.$) {
-				var iframeDocument = iframe.$.contentDocument;
-				var iframeBody = iframeDocument.body;
-				iframeBody.innerHTML = newData;
-			}
+		_sanitizeHTML: function (html) {
+			var sanitizedHtml = html
+				.replace(HTML_TAG_WITH_ON_ATTRIBUTE_REGEX, function (match) {
+					return match.replace(ON_ATTRIBUTE_REGEX, '');
+				})
+				.replace(ALERT_REGEX, '')
+				.replace(INNER_HTML_REGEX, '')
+				.replace(PHP_CODE_REGEX, '')
+				.replace(ASP_CODE_REGEX, '')
+				.replace(ASP_NET_CODE_REGEX, '');
+
+			return sanitizedHtml;
```

Comment:
Are we intentionally removing `_handleCodeMirrorChange`? It seems to update preview iframe. Won't this change result in preview never updating?

Reply:
Hello @markocikos,
Are you referring to this: https://github.com/liferay/liferay-ckeditor/pull/212/commits/799b5aff9116cb229cf7349c509d3f0ead9abdac#diff-8570f7b77fa9c9666a3f866cf502ba43abeb941c958aaf14683e3177eb60ecfeL186-L197

I found there were two `_handleCodeMirrorChange` methods that were exactly the same in the file. So I decided to remove one of them.
Let me know if I am misunderstanding.

Reply:
Oh wow, that's a nice cleanup then :smile: